bestiary/core/context.py

```diff
@@ -23,4 +23,7 @@
 import collections
 
 
-BestiaryContext = collections.namedtuple('BestiaryContext', ['user'])
+BestiaryContext = collections.namedtuple(
+    'BestiaryContext', ['user', 'job_id']
+)
+BestiaryContext.__new__.__defaults__ = (None, None)
```

Comment:
When the context is created by a job, a job identifier will be included, but when it is not created by a job, by default it will be None.

This change is based on SortingHat (https://github.com/chaoss/grimoirelab-sortinghat/blob/muggle/sortinghat/core/context.py#L29)

Reply:
Thanks! I completely forgot about it.